### DISCUSSION TITLE
Welcome to leetcode Discussions!

### DISCUSSION THREAD

**Discussion by haoel** [haoel/leetcode#242] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hardik302001** [comment]
hey, I also wanted to make a readme where I can see all my solutions in form of a table. Can you guide me on how to do it? I was looking around your code and found that you did it using scripts. Any resource from where I can implement the same for my repo.
Thanks.

**Comment by saikumar-pujari** [comment]
can you sort questions according to the easy,medium and hard level.

**Reply by chunyang-wen** [reply]
> can you sort questions according to the easy,medium and hard level.

I am sorry. The repo owner passed away. I am afraid it is still maintained.
